mitosheet/mitosheet/step_performers/graph_steps/graph_delete.py

```diff
@@ -55,26 +53,18 @@ def execute( # type: ignore
         }
 
     @classmethod
-    def transpile( # type: ignore
+    def transpile(
         cls,
         prev_state: State,
         post_state: State,
+        params: Dict[str, Any],
         execution_data: Optional[Dict[str, Any]],
-        graph_id: GraphID,
-        old_graph_tab_name: str
-    ) -> List[str]:
-        # Since we don't generate any code for graphs, don't return any transpiled code.
-        return []
+    ) -> List[CodeChunk]:
 
-    @classmethod
-    def describe( # type: ignore
-        cls,
-        graph_id: GraphID,
-        old_graph_tab_name: str,
-        df_names=None,
-        **params
-    ) -> str:
-        return f'Deleted {old_graph_tab_name}'
+        # Graph steps don't add any generated code to the analysis script. 
+        return [
+            EmptyCodeChunk(prev_state, post_state, params, execution_data)
```

Comment:
TODO: maybe I want to make it so the title and name are part of the empty code chunk, so you can still see it in the description of what occurred?